### DISCUSSION TITLE
Tracker TX timer with Smart Beaconing Off

### DISCUSSION THREAD

**Discussion by TechMindsYT** [lora-aprs/LoRa_APRS_iGate#138] [Q&A] [answered]
Hi there, 

Where can I change the TX timer when Smart Beaconing is turned off? I think it's currently TX'ing every 60 seconds but I would like to change this to 30 Seconds.

Thanks
Matt

**Comment by peterus** [comment, marked as answer]
This is the iGate repo. For the iGate it is sufficient to have like 15 oder 30 min 😉

**Reply by TechMindsYT** [reply]
oh damm, yes, you are right, wrong repo!! Thanks
